### DISCUSSION TITLE
개별 작업에 대한 건

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#15] [Ideas]
github desktop을 이용해서 간편하게 commit 및 push를 하는 것은 이제 익숙해졌습니다.
알맞은 폴더의 위치도 찾은 덕에 visual studio로 편하게 작업할수 있게 되었고요

**지금까지 느낀 바**로는 프로젝트를 개별적으로 작업을 하고 있어서 브랜치에서 새로운 파일(.cpp 또는 .h)을 생성하거나 기존 파일을 수정하는 정도입니다.
그러나 branch 또는 pull, patch 기능들을 제대로 맛볼려면 정말 **간단한 프로젝트라도 해야지 익숙해질 것** 같습니다.

느낀 바를 강조한 이유는 "이렇게 하고 싶다"가 아니기 때문입니다. 저는 명운장의 계획이 있을 것이라고 생각하고 있기때문에 그저 기다리고 있을 뿐

+제 개인 프로젝트를 뒤늦게 추가하기엔 파일 경로가 틀어져있어서 커밋이 어렵더라구요.(몰론 제가 못하는거일수도 있음)

**Comment by Woon-2** [comment]
빠르게 성과를 내고 계셔서 제가 속도를 못 따라가나 보네요... ㅋㅋㅋㅋ

이럴 때 증분으로 `branch`를 따로 예습해보시거나, cpp나 win32 심화 학습 하시고 싶은 부분 하시면 좋겠습니다.

>제 개인 프로젝트를 뒤늦게 추가하기엔...

이 부분을 해결해보면서 좀 더 `github desktop`에 적응해보셔도 좋습니다.

자급자족하고 남을 정도의 농작물을 수확하면, 다음 해 걱정이 없고, 타인에게 팔 수가 있습니다.
